src/abi_parse.cpp

```diff
@@ -91,8 +91,13 @@ void GccParser::build_scopes()
         }
 
         if (action_index < 0) {
-            throw std::runtime_error("invalid LSDA: call-site action offset "
-                                     "not found in action table");
+            std::cerr
```

Comment:
(don't need to fix now): Add a techdebt issue to sync error handling techniques with other modules (We will have this convo after term)